docs/source/reference/generic.rst

```diff
@@ -1,9 +0,0 @@
-NumPy-CuPy Generic Code Support
-===============================
-
-.. autosummary::
-   :toctree: generated/
-   :nosignatures:
-
-   cupy.get_array_module
-   cupyx.scipy.get_array_module
```

Comment:
Note: The functions are listed in "Code compatibility features" in `ndarray.rst`.